plugins/media-keys/gsd-media-keys-manager.c

```diff
@@ -1919,6 +1919,51 @@ do_config_power_action (GsdMediaKeysManager *manager,
         }
 }
 
+static gboolean
+supports_power_action (GsdMediaKeysManager *manager,
+                       GsdPowerActionType   action_type)
+{
+        const char *method_name = NULL;
+        GVariant *variant = NULL;
+        char *reply;
+
+        switch (action_type) {
+        case GSD_POWER_ACTION_SUSPEND:
+                method_name = "Suspend";
+                break;
+        case GSD_POWER_ACTION_SHUTDOWN:
+                method_name = "PowerOff";
```

Comment:
Wow, good catch. Indeed, this is broken on computers that do support any of these actions.